### DISCUSSION TITLE
2023년 03월 07일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#4] [General]
# 2023년 03월 07일

LOG 요즘 집중이 잘 안 된다. 오늘은 중간에 구현하다가 뛰쳐나왔다. 굿샷은 덥기도 했고.. 집중 좀 잘해보자. 브라운, 포이, 쥬니와 점심을 같이 먹었다. 연어 식당에 갔는데, 와사비를 빼고 먹기를 잘한 것 같다. 브라운이 스타트업 관련 이야기를…

https://cyma-s.github.io/2023-03-07/

**Comment by ezzanzzan** [comment]
난 공감 잘 해주는 베로가 좋아 성의없는 공감도 누군가에겐 공감 안해주는 것보단 나을 수 있다고 생각해요 🫶
